base/char.jl

```diff
@@ -134,7 +141,7 @@ function decode_overlong(c::Char)
 end
 
 """
-    decode_overlong(c::AbstractChar)
+    decode_overlong(c::AbstractChar) -> UInt32
```

Comment:
Is this return type for all `AbstractChar` or only for `Char`?

Reply:
The result is a code point which there's no reason to represent as anything but a `UInt32`.